### PR TITLE
feat(docs): add react console documentation

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
Convenience link: https://patternfly-org-pr-3366-v4.surge.sh/v4/extensions/react-console/

### CODE REVIEW

**Review by dlabaj — APPROVED** [review]
LGTM

**Review by evwilkin — APPROVED** [review]
LGTM 👏

**Comment by KKoukiou** [comment]
@wise-king-sullyman the page is missing all the react-console CSS here

**Review by KKoukiou — COMMENTED** [review]
https://github.com/patternfly/patternfly-org/pull/3366#issuecomment-1410276404

**Review by evwilkin — APPROVED** [review]
LGTM

File: packages/v4/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "@patternfly/react-catalog-view-extension": "4.95.2",
     "@patternfly/react-docs": "5.103.66",
     "@patternfly/react-log-viewer": "4.87.100",
-    "@patternfly/react-topology": "^4.91.26",
```
Comment (KKoukiou) [inline packages/v4/package.json:25]: This does not look correct. You don't want to remove topology in place of console

**Review by KKoukiou — CHANGES_REQUESTED** [review]

File: packages/v4/package.json (hunk 1 above)
Comment (wise-king-sullyman) [reply packages/v4/package.json:25]: Ah yep, not sure if I mixed something up resolving merge conflicts during a rebase or what. Great catch.

**Review by KKoukiou — APPROVED** [review]

**Review by evwilkin — DISMISSED** [review]
LGTM

**Review by evwilkin — APPROVED** [review]
LGTM

**Pull request merged**